lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskQuerySqlProvider.java

```diff
@@ -339,6 +449,30 @@ private static String commonTaskObjectReferenceWhereStatement() {
         + "</if>";
   }
 
+  private static String commonTaskObjectReferenceWhereStatementOracle() {
+    return "<if test='objectReferences != null'>"
+        + "AND (<foreach item='item' collection='objectReferences' separator=' OR '> "
+        + "<if test='item.company != null'>t2.POR_COMPANY = #{item.company} </if>"
+        + "<if test='item.system != null'> "
+        + "<if test='item.company != null'>AND</if> "
+        + "t2.POR_SYSTEM = #{item.system} </if>"
+        + "<if test='item.systemInstance != null'> "
+        + "<if test='item.company != null or item.system != null'>AND</if> "
+        + "t2.POR_INSTANCE = #{item.systemInstance} </if>"
+        + "<if test='item.type != null'> "
+        + "<if test='item.company != null or item.system != null or item.systemInstance != null'>"
+        + "AND</if> "
+        + "t2.POR_TYPE = #{item.type} </if>"
+        + "<if test='item.value != null'> "
+        + "<if test='item.company != null or item.system != null "
+        + "or item.systemInstance != null or item.type != null'>"
+        + "AND</if> "
+        + "t2.POR_VALUE = #{item.value} "
+        + "</if>"
+        + "</foreach>)"
+        + "</if>";
```

Comment:
this is a direct consequence of `commonTaskWhereStatementOracle`, right?

Reply:
yes it's removed